src/interfaces/coral_web/src/styles/fonts.css

```diff
@@ -1,6 +1,6 @@
 @font-face {
   font-family: 'CohereText';
-  src: url('https://fonts.cohere.com/CohereVariable.woff2') format('woff2'),
+  src: url('https://fonts.cohere.com/CohereText-Regular.woff2') format('woff2'),
```

Comment:
Just to clarify, do we need to move this font to /public/fonts too, or does it work well as is?

Reply:
We can't move the fonts back into the codebase because they're not open source unfortunately. :( But at least if these fail we can fallback on a system font vs for icons there's no fallback